dashboard/main.go

```diff
@@ -62,7 +65,7 @@ func main() {
 	gosms.InitWorker(modems, bufferSize, bufferLow, loaderTimeout, loaderCountout, loaderTimeoutLong)
 
 	log.Println("main: Initializing server")
-	err = InitServer(serverhost, serverport)
+	err = InitServer(serverhost, serverport, serverusername, serverpassword)
```

Comment:
sorry, not in touch with Golang off late. What happens if username and passwords are empty? Do you still get a prompt and requires to hit enter?